Create ui-c8y-1018-503-150-cumulocity-io-t-sso-ui-bug-in-2024-9-version-8412.md (#3093)
SoftwareAG/c8y-docs: content/change-logs/platform-services/ui-c8y-1018-503-150-cumulocity-io-t-sso-ui-bug-in-2024-9-version-8412.md

---
date: ""

> title: Cumulocity IoT SSO UI bug in 2024.9version (#8412)

Reviewer: ```suggestion
title: Fixed an issue where the SSO form would break when selecting multiple default global roles, default applications, or both
```

> title: Fixed an issue where the SSO form would break when selecting multiple default global roles, default applications, or both

Reviewer 2: ```suggestion
title: All fields in single sign-on configuration form accessible when selecting multiple options
```

product_area: Platform services
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:

>   - value: component-0UgqXH1Ys

Reviewer 2: ```suggestion
  - value: q3kclF6pO
```

>     label: Administration

Reviewer 2: ```suggestion
    label: Authentication
```

build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-62499
version: 1018.503.150
---

> Cumulocity IoT SSO UI bug in 2024.9version (#8412)

Reviewer: ```suggestion
Previously, when users attempted to configure Single Sign-On settings and selected multiple default global roles, multiple default applications, or a combination of both, the form would extend beyond the visible area, making some fields inaccessible. Users can now properly view and complete all form fields regardless of their selection.
```

> Previously, when users attempted to configure single sign-on settings and selected multiple default global roles, multiple default applications, or a combination of both, the form extended beyond the visible area, making some fields inaccessible. Users can now properly view and complete all form fields regardless of their selection.

Reviewer 2: ```suggestion
Previously, when users configured single sign-on settings and selected multiple default global roles, multiple default applications, or a combination of both, the form extended beyond the visible area, making some fields inaccessible. Users can now properly view and edit all fields regardless of their selection.
```

> Previously, when users attempted to configure Single Sign-On settings and selected multiple default global roles, multiple default applications, or a combination of both, the form would extend beyond the visible area, making some fields inaccessible. Users can now properly view and complete all form fields regardless of their selection.

Reviewer 2: ```suggestion
Previously, when users attempted to configure single sign-on settings and selected multiple default global roles, multiple default applications, or a combination of both, the form extended beyond the visible area, making some fields inaccessible. Users can now properly view and complete all form fields regardless of their selection.
```
